backend/django/core/views/api_annotate.py

```diff
@@ -169,6 +169,10 @@ def annotate_data(request, data_pk):
     label = Label.objects.get(pk=request.data["labelID"])
     labeling_time = request.data["labeling_time"]
 
+    label_reason = ""
+    if "labelReason" in request.data:
+        label_reason = request.data["labelReason"]
```

Comment:
This can be shortened to

`label_reason = request.data.get("labelReason", "")`

Reply:
This actually appears in a few spots further down so you can change in all those places please